connectors/http-json/element-templates/http-json-connector.json

```diff
@@ -284,6 +372,38 @@
         "equals": "oauth-client-credentials-flow"
       }
     },
+    {
+      "label": "Authentication Output Headers",
+      "description": "Headers that will parsed from authentication response body and put in request. Details in the <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/rest/#rest-connector-custom-authentication\" target=\"_blank\">documentation</a>",
+      "group": "authentication",
+      "type": "Text",
+      "feel": "required",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.outputHeaders"
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "credentialsInBody"
+      },
+      "optional": true
+    },
+    {
+      "label": "Authentication Output Body",
+      "description": "Body properties that will parsed from authentication response body and put in request. Details in the <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/rest/#rest-connector-custom-authentication\" target=\"_blank\">documentation</a>",
+      "group": "authentication",
+      "type": "Text",
+      "feel": "required",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.outputBody"
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "credentialsInBody"
+      },
+      "optional": true
+    },
```

Comment:
Just a note: if I add some value here for `Authentication Output Body`, and then select `GET` as method in the _HTTP Endpoint_ section, it will try to execute a GET method with a body which will result in a `java.lang.IllegalStateException: Apache HTTP client does not support GET requests with content.` exception. As far as I know we cannot add constraints to dropdown values, but if there is a way to do it, maybe it would be something worth considering. We could create a ticket about this, as a suggestion for future development for the template engine. Do you also think it is something notable? Do you know any way to add a constraint to a specific dropdown value?

Reply:
If we do not have a solution for this right now, all changes are good from my side.